message/association-setup-request.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2019-2020 go-pfcp authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+package message
+
+import "github.com/wmnsk/go-pfcp/ie"
+
+// AssociationSetupRequest is a AssociationSetupRequest formed PFCP Header and its IEs above.
+type AssociationSetupRequest struct {
+	*Header
+	NodeID *ie.IE
+	IEs    []*ie.IE
+}
+
+// NewAssociationSetupRequest creates a new AssociationSetupRequest.
+func NewAssociationSetupRequest(nid *ie.IE, IEs ...*ie.IE) *AssociationSetupRequest {
```

Comment:
Don't capitalize the name of parameter.

```suggestion
func NewAssociationSetupRequest(nid *ie.IE, ies ...*ie.IE) *AssociationSetupRequest {
```